### ISSUE TITLE
@mentions and #tags fail color contrast accessibility

### ISSUE THREAD

**Issue by Phizzirrat** [tuskyapp/Tusky#1831]
The light blue text for mentions and hashtags fails accessibility guidelines on light version of tusky. Color contrast is less than 2.2:1 (requires 4.5:1 or better).

Suggestion darken mention/hashtag and lighten background.

* * * *
- Tusky Version: 11
- Android Version: 9
- Android Device: Moto X4
- Mastodon instance (if applicable):

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
